neuro_extras/main.py

```diff
@@ -1048,7 +1050,7 @@ async def _create_builder_container(
         return neuro_api.Container(
             image=neuro_api.RemoteImage(
                 name="gcr.io/kaniko-project/executor",
-                tag="latest",
+                tag="v1.1.0",
```

Comment:
thanks for this! By the way: recently, I was struggling with finding the avail tags -- it's not dockerhub. Where can I get the list of available tags?

Reply:
I believe a shortlist is only here https://github.com/GoogleContainerTools/kaniko/releases , full list is here: gcr.io/kaniko-project/executor